docs/release-notes.rst

```diff
@@ -46,6 +46,32 @@ The default docker0 bridge IP has the address **172.17.0.1**. If the docker0 bri
 
 ------
 
+**Hostname vCMP HA peer login during unassign VLANs does not update the login color to Green/Yellow in the side menu**
+
+On a vCMP Host, if a user clicks the **L2-L3 Network Management > vCMP Guest** and selects a vCMP Guest and moves a few VLANs from **Selected** menu to **Available** menu and clicks **Submit**, F5 ACI ServiceCenter logs into the vCMP Guest if it is not already logged in. In this case, the side menu does not show the Green/Yellow color indicator correctly. 
```

Comment:
...if a user clicks **L2-L3 Network Management > vCMP Guest**, selects a vCMP Guest, moves a few VLANs from the **Selected** menu to the **Available** menu, and then clicks **Submit**,